### PR TITLE
Support multiple audio sources

### PR DESCRIPTION
**Pull request by auriiolym**
**Description:**
This turned out to be much more than just support for multiple audio sources. Besides the changes listed below, several architectural changes were applied.

**Changes proposed:**
- Allow multiple audio sources per audio room.
- The readme file has been updated to include the API description.
- An audio source can now have a custom position and rotation, irrespective of its entity's position and rotation. I needed this for my own project.
- Remove linting warnings.
- Add test suite.
- Add optional wireframe visualization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,4 +188,4 @@ document.querySelector('a-resonance-audio-src').components['resonance-audio-src'
 [etiennepinchon/aframe-resonance]: https://github.com/etiennepinchon/aframe-resonance
 [resonance-audio-link]: https://developers.google.com/resonance-audio/
 
-[gh-pages-link]: https://digaverse.github.io/aframe-resonance-audio-component/
+[gh-pages-link]: https://digaverse.github.io/aframe-resonance-audio-componen
```
Comment (mkungla) [inline README.md]: here is typo, deleted line was correct

File: package.json [hunk 2]
```diff
@@ -24,16 +24,44 @@
...
+    "karma": "^2.0.4",
+    "karma-browserify": "^5.3.0",
+    "karma-chai-shallow-deep-equal": "0.0.4",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sinon-chai": "^1.3.4",
+    "log-suppress": "^1.0.3",
```
Comment (mkungla) [inline package.json]: Shouldn't use `log-suppress` since it is stale repository and requires `"engines": { "node": "5.0.0" }`

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aframe-resonance-audio-component",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (mkungla) [inline package.json]: revert to version `0.1.0` so that after merge could increment version when publishing new release

**Comment by mkungla** [comment]
In general nice work you have done here, getting this merged enables to publish it to npm as well

File: README.md (hunk 1 above)
Comment (auriiolym) [reply README.md]: No idea how this happened. Will fix.

File: package.json (hunk 2 above)
Comment (auriiolym) [reply package.json]: Okay, I've manually disabled console.log. Else the console is cluttered with irrelevant information during testing.

**Comment by mkungla** [comment]
Thanks!

**Comment by mkungla** [comment]
@auriiolym whats your slack username in A-Frame slack?

**Pull request merged**